src/main/java/servlets/BillingConfig.java

```diff
@@ -58,6 +61,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       configText = "Error finding " + vendorID + "'s configuration";
     }
 
+    /** 
+     * TODO: @cade, eventually you will want to pass this access token into the 
+     * updateSheets() method in the SheetsConverter class.
+     */
+    HttpSession session = request.getSession();
+    String accessToken = session.getAttribute("accessToken").toString();
+
+    SheetsConverter sheet = new SheetsConverter(); 
+    try {
+      sheet.writeToSheet(accessToken);
+    } catch (GeneralSecurityException e) {
+      // TODO: @cade Figure out how you want to handle this error.
```

Comment:
Maybe an error alert on this exception? A possible suggestion for this.

Reply:
Yea, this is just something @cadecrandall and @Vincent need to get finalized. I don't know how you two plan on handling errors.